docs/examples/sudoers.d/fty_00_base.in

```diff
@@ -34,10 +34,10 @@ Cmnd_Alias BIOS_LOGHOST_RSYSLOG   = @libexecdir@/@PACKAGE@/loghost-rsyslog
 Cmnd_Alias BIOS_MYSQLDUMP   = /usr/bin/mysqldump
 Cmnd_Alias BIOS_DMF         = /usr/bin/bios-dmf
 Cmnd_Alias BIN_SYSTEMCTL    = /bin/systemctl
-Cmnd_Alias BIN_IP_ROUTE     = /bin/ip route
+Cmnd_Alias BIN_FTY_ROUTE    = @libexecdir@/@PACKAGE@/fty-route
```

Comment:
Also, in the spirit of variables named above, although clearly not done everywhere "cleanly", `BIN_` prefix is for OS-provided tools, and `BIOS_` (or maybe `FTY_`) is for wrappers and tools from our project, so these differences are better visible for maintenance etc. (e.g. to estimate "is it our direct failure that tool X does not work as expected?").